frontend/components/product-list/sections/FilterableProductsSection/FacetsDrawer.tsx

```diff
@@ -0,0 +1,318 @@
+                              onSelect={setCurrentFacet}
+                              refinedCount={refinedCount}
+                           />
+                        );
+                     })}
+                     {facets.map((facet) => {
+                        return (
+                           <FacetPanel
+                              key={facet}
+                              attribute={facet}
+                              isOpen={facet === currentFacet}
+                           />
+                        );
+                     })}
+                  </Box>
+               </Box>
+               <Box>
+                  <Divider borderColor="gray.300" />
+                  <HStack p="2">
+                     {facets.map((facet) => {
+                        return (
+                           <ClearFacetButton
+                              key={facet}
+                              attribute={facet}
+                              isVisible={facet === currentFacet}
+                           />
+                        );
+                     })}
+                     <ClearAllButton isVisible={currentFacet == null} />
+                     <Button
+                        w="50%"
+                        colorScheme="blue"
+                        onClick={() => {
+                           if (currentFacet == null) {
+                              onClose();
+                           } else {
+                              setCurrentFacet(null);
+                           }
+                        }}
+                     >
+                        Apply
+                     </Button>
+                  </HStack>
+               </Box>
+            </Flex>
+            <Box flexGrow={1} h="100vh" onClick={onClose} />
+         </Flex>
+      </Portal>
+   );
+}
+
+type ClearFacetButtonProps = {
+   attribute: string;
+   isVisible: boolean;
+};
+
+function ClearFacetButton({ attribute, isVisible }: ClearFacetButtonProps) {
+   const { items } = useRefinementList({ attribute });
+   const { refine } = useClearRefinements({
+      includedAttributes: [attribute],
+   });
+   const refinedCount = items.filter((item) => item.isRefined).length;
+   if (!isVisible) {
+      return null;
+   }
+   return (
+      <Button
+         w="50%"
+         variant="outline"
+         fontWeight="normal"
+         disabled={refinedCount === 0}
+         onClick={() => refine()}
+      >
+         {refinedCount > 1
+            ? `Clear ${refinedCount} filters`
+            : refinedCount === 1
+            ? `Clear ${refinedCount} filter`
+            : 'Clear'}
+      </Button>
+   );
+}
+
+type ClearAllButtonProps = {
+   isVisible: boolean;
+};
+
+function ClearAllButton({ isVisible }: ClearAllButtonProps) {
+   const { refine } = useClearRefinements();
+   if (!isVisible) {
+      return null;
+   }
+   return (
+      <Button w="50%" variant="outline" onClick={() => refine()}>
```

Comment:
```suggestion
      <Button w="50%" variant="outline" onClick={refine}>
```

Reply:
I'm doing this on purpose, since `onClick` passes `event` as an argument, and if for some reason the `refine` method implementation is changed to receive an argument this could potentially introduce an unexpected bug